### DISCUSSION TITLE
Can I import JS SDK in Sandpack?

### DISCUSSION THREAD

**Discussion by Zih0** [codesandbox/sandpack#599] [Q&A] [answered]
I tried to import google maps sdk.
but it didn't work.
help me please🙏

https://codesandbox.io/s/sandpack-monaco-editor-forked-kcr96c?file=/src/App.tsx

https://codesandbox.io/embed/github/googlemaps/js-samples/tree/sample-map-simple

**Comment by danilowoz** [comment, marked as answer]
Unfortunately, you can't load external scripts by injecting them into the head. This's a hard limitation on Sandpack, as described on https://github.com/codesandbox/sandpack/issues/44, so you need to use `externalResource` to load these scripts.

Something like this: https://codesandbox.io/s/sandpack-monaco-editor-forked-g1skmw?file=/src/App.tsx:0-9

I wasn't able to make it fully work, but maybe you have another idea why. Hope it helps :)

**Reply by Zih0** [reply]
Thanks for the response! 😄
